### ISSUE TITLE
No graph output

### ISSUE THREAD

**Issue by davidlenz** [datasciencecampus/pygrams#369]
**Describe the bug**

When running the command

 `python pygrams.py -dh publication_date -uc=out-mdf-0.05-200501-201822 -o graph`

 i would expect a graph html-output in the graph folder, however there is none. 
Works well with all other outputs, e.g. wordcloud and multiplot. No error thrown or anything.

**To Reproduce**
1. Create tf-idf matrices: `python pygrams.py -dh publication_date -ds=USPTO-random-100000.pkl.bz2`
2. `python pygrams.py -dh publication_date -uc=out-mdf-0.05-200501-201822 -o graph`
3. check in outputs folder `out-mdf-0.05-200501-201822`
4. See no html file

**Expected behavior**
Have .html graph file in graph folder

**Screenshots**
![grafik](https://user-images.githubusercontent.com/25869733/83014599-4dec8d00-a01f-11ea-94ff-4b41b0c7d22b.png)

**Desktop (please complete the following information):**
 - OS: Win10

**Additional context**
-

**Comment by mshodge** [comment]
Investigating using fresh repo and environment. Will report back.

**Comment by mshodge** [comment]
Hi @davidlenz unfortunately this is old code that we did not remove. We used to use NetworkX's Force Directed Graph (FDG) but there were issues with combining the D3/JS element of this with our code. As a result, we stopped this feature. I will remove reference to it from documents and arguments. Sorry if you thought this feature was useful.

I will make a ticket to replace this feature with something more Python friendly. Perhaps an interactive chord diagram instead, which shows the links between terms, albeit not with the force directed component.
